integration-tests/tests/init/captureErrors.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Test init</title>
+
+	<%- renderAgent({captureErrors: false}) %>
+</head>
+<body>
+	<p>Attributes</p>
```

Comment:
I like to use different html content so differentiating tests on browserstack video is easier.